File: src/main/java/com/lambdaschool/starthere/services/StoryService.java
package com.lambdaschool.starthere.services;

import com.lambdaschool.starthere.models.Story;

import java.util.List;

public interface StoryService
{
    List<Story> findAll();

    Story findQuoteById(long id);

    List<Story> findByUserName(String username);

    void delete(long id);

    Story save(Story story);
}

File: kotlinstory/src/main/kotlin/com/example/kotlinstory/services/StoryService.kt
package com.example.kotlinstory.services

import com.example.kotlinstory.models.Story
//todo may not use
interface StoryService {
    fun findAll(): List<Story>

    fun findStoryById(id: Long): Story

    fun findByUserName(username: String): List<Story>

    fun delete(id: Long)

    fun save(quote: Story): Story

    fun update(id:Long, quote: Story)
}
